db/init/add_dataset_db.sql

```diff
@@ -25,19 +25,36 @@ CREATE TABLE public.spatial
     "NAME_ENGL" varchar(100),
     "CNTR_CODE" char(2),
     "LEVL_CODE" levl,
-    geometry geometry(Geometry,4326)
+    ds_id int,
+    geometry geometry(Geometry,3035)
 );
 
 CREATE TABLE public.data
 (
-    index int PRIMARY KEY,
+    index SERIAL PRIMARY KEY,
     "time" timestamp without time zone,
     fields jsonb,
-    variable varchar(100),
+    variable varchar(200),
+    unit varchar(200),
     value double precision,
     ds_id int,
     "FID" varchar(200),
     dt double precision,
     z double precision,
     "Raster" boolean
```

Comment:
Be coherent in the naming of your variables (for example, all variable names in lowercase letters). Please, avoid using quotes, as this can cause problems like the one we had here:
https://github.com/enermaps/enermaps/pull/176#issuecomment-811033866

Reply:
Quotes are needed for uppercase column names (which are indeed not encouraged) in Postgres queries. FID (which stands for Feature Identifier) is usually written with uppercases, so it's a bit difficult to find a convention that applies to every naming here...

Reply:
I think that it would be better to put aside the "real world convention" that puts FID in uppercase letters, because the lambda user is not going to read inside your DB. On the contrary, there is a much higher risk that someone else than me will try to execute your test script and comes across the same problem than me. I had to look for two hours to find where it came from...
So I suggest you to put everything in lowercase letters.

Reply:
Ok!